maven/jdk11/Dockerfile.nanoserver

```
@@ -1,2 +1,18 @@
-# Use https://github.com/jenkinsci/jnlp-agents as base for now
-FROM jenkins/jnlp-agent-maven:windows-nanoserver-jdk11 AS jnlp
+# escape=`
+FROM mcr.microsoft.com/windows/servercore:1809 as core
+FROM jenkins/inbound-agent:jdk11-nanoserver-1809
+
+# ProgressPreference => Disable Progress bar for faster downloads
+SHELL ["pwsh.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# https://github.com/StefanScherer/dockerfiles-windows/tree/master/golang-issue-21867
+COPY --from=core C:/windows/system32/netapi32.dll C:/windows/system32/netapi32.dll
+
+ARG MAVEN_VERSION=3.8.6
+RUN Invoke-WebRequest -Uri "https://archive.apache.org/dist/maven/maven-3/${env:MAVEN_VERSION}/binaries/apache-maven-${env:MAVEN_VERSION}-bin.zip" -OutFile ${env:TEMP}/apache-maven.zip ; `
+  Expand-Archive -Path "${env:TEMP}/apache-maven.zip -Destination" C:\tools ; `
+  Remove-Item ${env:TEMP}/apache-maven.zip ;
+
+ENV JAVA_HOME ${JAVA_HOME}
+ENV MAVEN_HOME "C:/tools/apache-maven-${MAVEN_VERSION}"
+ENV PATH="${WindowsPATH};${ProgramFiles}\PowerShell;${JAVA_HOME}\bin;${MAVEN_HOME}\bin"
```

There is a mix between "/" and "\", it seems to me that at least the last path in $PATH will cause issue as it's using both of them.

```suggestion
ENV MAVEN_HOME "C:\tools\apache-maven-${MAVEN_VERSION}"
ENV PATH="${WindowsPATH};${ProgramFiles}\PowerShell;${JAVA_HOME}\bin;${MAVEN_HOME}\bin"
```

or eventually (?)
```suggestion
ENV MAVEN_HOME "C:/tools/apache-maven-${MAVEN_VERSION}"
ENV PATH="${WindowsPATH};${ProgramFiles}\PowerShell;${JAVA_HOME}\bin;${MAVEN_HOME}/bin"
```

---

Currently checking